lhotse/recipes/voxceleb.py

```diff
@@ -89,33 +93,76 @@ def download_voxceleb1(
 
     :param target_dir: Pathlike, the path of the dir to store the dataset.
     :param force_download: bool, if True, download the archive even if it already exists.
+    :return: the path to downloaded and extracted directory with data.
+    """
+
+    return _dowload_voxceleb(
+        voxceleb_name="VoxCeleb1",
+        part_urls=VOXCELEB1_PARTS_URL,
+        part_suffix="vox1_dev_wav_part",
+        dev_zip_name="vox1_dev_wav.zip",
+        test_zip_name="vox1_test_wav.zip",
+        target_dir=target_dir,
+        force_download=force_download,
+    )
+
+
+def _dowload_voxceleb(
```

Comment:
Good idea to avoid code duplication. Nit: put this method at the top before the `download_voxceleb1()` function.

Reply:
Done